react/features/connection-indicator/components/ConnectionIndicator.js

```diff
@@ -198,17 +175,45 @@ class ConnectionIndicator extends Component {
      */
     render() {
         return (
-            <div
-                className = 'connection-indicator indicator'
-                ref = { this._setRootElement }>
-                <div className = 'connection indicatoricon'>
-                    { this._renderIcon() }
+            <AKInlineDialog
+                content = { this._renderStatisticsTable() }
+                isOpen = { this.state.showStats }
+                onClose = { this._onStatsClose }
+                position = { this.props.statsPopoverPosition }>
```

Comment:
@yanas, you can directly play with positioning on this line. Maybe you want to try another position other than on the left? Maybe bottom/top? You might encounter z-indexing issues but those should be fixable. What you seem to be expecting is the old jitsipopover behavior that uses jquery to nudge the position to where you want and nudge the view to make it fit in the viewport. However, I can't get that behavior with AtlasKit but maybe you can figure out a way and I'm overlooking it. https://atlaskit.atlassian.com/components/inline-dialog